internal/pkg/cli/cli.go

```diff
@@ -181,3 +184,40 @@ type storeReader interface {
 	archer.ApplicationLister
 	archer.ApplicationGetter
 }
+
+// returns true if error type is stack set not exist.
+func stackSetNotExistErr(err error) bool {
```

Comment:
nit: prefix with `is`
```suggestion
func isStackSetNotExistsErr(err error) bool {
```